hackertracker.xcodeproj/project.pbxproj

```diff
@@ -422,6 +424,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "org.beezle.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
+				SWIFT_VERSION = 3.0;
```

Comment:
Just a heads up Xcode 8.2.1 should be updating this value to 3.0.2, even though it makes 0 difference for the most part (just a few Swift bugs fixed).